src/map.h

```diff
@@ -1311,6 +1311,13 @@ class map
          * called the last time.
          */
         void restock_fruits( const tripoint &p, const time_duration &time_since_last_actualize );
+        /**
+         * Try to restore a cut static plant to normal.
+         * @param p Place to restore
+         * @param time_since_last_actualise Time since this function has been
+         * called the last time.
```

Comment:
> Time since this function has been called the last time.

Oh, I see where the misconception came from. The function above has same (wrong) comment.
The correct description would be:
> Time since the map segment was last loaded